### DISCUSSION TITLE
Listado de proveedores

### DISCUSSION THREAD

**Discussion by gadc1996** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#22] [Q&A]
¿Como podemos obtener un listado completo de los proveedores?

**Comment by AndresRamos** [comment]
No hay un request específico para eso, pero utilizando el request `BuscarClientesRequest` puedes buscar los proveedores utilizando el parámetro `sqlQuery`.

```json
{
  "$type": "BuscarClientesRequest",
  "model": {
    "sqlQuery": "CTIPOCLIENTE IN (3, 2)"
  },
  "options": {}
}
```

En CONTPAQi Comercial los clientes y proveedores comparten la misma tabla y los puedes diferenciar por el campo `CTIPOCLIENTE` que es de tipo `int` donde: 1 = Cliente, 2 = Cliente/Proveedor, 3 = Proveedores.

Si solo buscas proveedores entonces debes buscar los registros que son de tipo 2 y 3.

Pruébalo y déjame saber si te funciono.
